_episodes/04-lists.md

```diff
@@ -195,22 +205,37 @@ does not.
 > {: .language-python}
 >
 > ~~~
-> ['pepper', 'zucchini', 'onion']
+> ['lettuce', 'lettuce', 'peppers', 'zucchini', 'squash', 'basil']
 > ~~~
 > {: .output}
 >
+> To reference a specific basket on a specific shelf, you use two indexes. The first
+> index represents the row (from top to bottom) and the second index represents
+> the specific basket (from left to right).
+> [![x is now shown as a two-dimensional grid, with each basket labeled according to
+> its index in the nested list. The first index is the row number and the second
+> index is the basket number, so x[1][4] represents the basket on the far right
+> side of the second row (basket 4 on row 1): parsley](../fig/04_groceries_x00.png)]
```

Comment:
The basket index given in the text does not match up with where I see parsley in the image.